.github/workflows/on-pullrequest.yml

```diff
@@ -45,27 +43,29 @@ jobs:
         with:
           fetch-depth: 0w
 
-      - name: Get changes
-        id: changes
+      - name: Get version changes
+        id: versions
```

Comment:
This is a refactor of the logic to check if it's a PR in the script, that way we avoid having to add the always() logic in the subsequent jobs, instead of line 35

Reply:
See it in action here:
on main:
https://github.com/andonyns/Meadowlark/actions/runs/6947706833
on PR:
https://github.com/andonyns/Meadowlark/actions/runs/6947715142